src/doom/p_mobj.c

```diff
@@ -803,6 +803,12 @@ void P_SpawnMapThing (mapthing_t* mthing)
 	bit = 1;
     else if (gameskill == sk_nightmare)
 	bit = 4;
+    else if (gameskill < sk_baby)
```

Comment:
Why don't you just replace the `else` part?

Reply:
When I originally wrote the proposed code I wanted to avoid change in the default case just in case there was any problem in the replacement.

Reply:
Yeah, based on what we know now, we could probably just change the else branch directly.